apps/server/src/infra/identity-management/keycloak-configuration/service/keycloak-configuration.service.integration.spec.ts

```diff
@@ -3,10 +3,10 @@ import KeycloakAdminClient from '@keycloak/keycloak-admin-client-cjs/keycloak-ad
 import AuthenticationExecutionExportRepresentation from '@keycloak/keycloak-admin-client/lib/defs/authenticationExecutionExportRepresentation';
 import AuthenticationFlowRepresentation from '@keycloak/keycloak-admin-client/lib/defs/authenticationFlowRepresentation';
 import { EntityManager } from '@mikro-orm/mongodb';
+import { systemEntityFactory } from '@modules/system/testing';
```

Comment:
Hier auch: Der Import widerspricht der eslint-Regel "infra-modules may NOT import from apps, https://github.com/core, @modules, or migrations". Ist das gewollt?